### PR TITLE
feat(ObjectPage): Add 'Pin-Header' button

### LINKED ISSUE #248: ObjectPage: Collection of known issues/limitations
**List of issues with [Severity]** 
- [ ] [MED]Wrong scroll position when item of IconTabBar is clicked. The scroll offset value has to be calculated, when the header is changing from expanded to collapsed state.
- [ ] [LOW] Native scrollbar is still visible in some cases.
- [ ] [LOW] The Title of the first SubSection don't hide if the first element inside the ObjectPage is not a SubSection (https://github.com/SAP/ui5-webcomponents-react/issues/25)

**Reproduce 1**
1. ObjectPage with expanded header and default mode.
2. Press item in the item tab bar in order to scroll to the selected section.

**Reproduce 2**
- Scrolling with mouse near left border of scrollbar.
Double scroll bars

### LINKED ISSUE #334: Adding "Pin" Button to ObjectPage Component
**Is your feature request related to a problem? Please describe.**
In the Fiori 3.0 designs, the ObjectPage component includes a button alongside the expand/collapse header button to allow a user to "pin" the header in an expanded state. We would like to use this functionality to follow Fiori.

**Describe the solution you'd like**
The addition of a button (or ability to add a button ourselves) that allows a user to "pin" the header content of the ObjectPage component in an expanded state. This would allow a user to continue to scroll through content without pushing the header content into a collapsed state.

**Additional context**
Thank you!

### PR DESCRIPTION
**Pull request by MarcusNotheis**
This PR includes a full refactoring of the ObjectPage using native CSS capabilities instead of custom scroll handlers. This might lead to problems in IE11 and needs to be tested as well.

The Anchor Bar was replaced with the UI5 Web Component TabContainer in order to improve accessibility and RTL. --> #345 

Closes #334 
Closes #248

### CODE REVIEW

**Comment by clemai** [comment]
Minor bugs:

- Expand/Collapse button directs in the wrong direction when scrolling down: 

image

- 1.) Click to sub section -> OP scrolls to sub section 2.) Click to section of sub section -> OP doesn't scroll.

- Little flickering of object page header when scrolling

**Comment by MarcusNotheis** [comment]
The mentioned bugs should be now fixed - ready for re-review.
The header behaviour is now similar to the UI5 Object Page: When header is expanded in a scrolled down state, it will collapse on next scroll.

File: packages/main/src/components/ObjectPage/useObserveHeights.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      headerContentResizeObserver.observe(headerContentRef.current);
+    }
+    return () => {
+      headerContentResizeObserver.disconnect();
+    };
+  }, [headerContentRef.current, setHeaderContentHeight]);
+
+  const anchorBarHeight = anchorBarRef.current?.offsetHeight ?? 33;
+
+  // const totalHeaderHeight =
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/useObserveHeights.ts]: still needed?

File: packages/base/src/Scroller/Scroller.tsx [hunk 2]
```diff
@@ -17,6 +17,17 @@ export const Scroller = forwardRef((props: Props, ref: RefObject<IScroller>) =>
 
   const elements = useRef([]);
 
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (MarcusNotheis) [reply packages/base/src/Scroller/Scroller.tsx]: done

File: packages/main/src/components/ObjectPage/useObserveHeights.ts (hunk 1 above)
Comment (MarcusNotheis) [reply packages/main/src/components/ObjectPage/useObserveHeights.ts]: no :)

File: packages/main/src/components/ObjectPage/ObjectPage.jss.ts [hunk 3]
```diff
@@ -5,76 +5,65 @@ const styles = {
...
-  },
-  outerContentContainer: {
-    width: '100%',
-    height: '100%',
-    overflow: 'hidden'
-  },
-  contentScrollContainer: {
-    position: 'relative'
+    '&::-webkit-scrollbar': {
+      backgroundColor: '#ffffff',
```
[the remark this answers is not in the document]
Comment (MarcusNotheis) [reply packages/main/src/components/ObjectPage/ObjectPage.jss.ts]: done

**Pull request merged**